lib/Doctrine/DBAL/Platforms/SQLAnywherePlatform.php

```diff
@@ -551,7 +551,7 @@ public function getDropIndexSQL($index, $table = null)
 
         if (! is_string($index)) {
             throw new InvalidArgumentException(
-                'SQLAnywherePlatform::getDropIndexSQL() expects $index parameter to be string or ' . Index::class . '.'
+                sprintf('SQLAnywherePlatform::getDropIndexSQL() expects $index parameter to be "string" or "%s".', Index::class)
```

Comment:
Should it be "…or an instance of %s", Index::class? Classes probably shouldn't be enclosed in quotes since they are symbols, not strings.